lib/sb3.js

```diff
@@ -164,8 +164,10 @@ const extensions = function (list) {
 
 const metadata = function (meta) {
     let obj = {};
-    if (meta.origin) {
-        obj.origin = meta.origin;
+    for (const key of ['semver', 'vm', 'agent', 'origin']) {
+        if (meta[key]) {
```

Comment:
```suggestion
        if (Object.hasOwn(meta, key)) {
```

This way will copy any false-like values, if they're present, instead of ignoring them. Probably not a big deal right now since none of those fields allow for such values, but we might add more fields to this list in the future and it would be easy to miss this at that time.

Reply:
@cwillisf Makes sense, applied